docs/source/llms/chat-model-guide/index.rst

```diff
@@ -0,0 +1,802 @@
+Tutorial: Custom GenAI Models using ChatModel
+=============================================
+
+The rapidly evolving landscape of Generative Artificial Intelligence (GenAI) presents exciting opportunities and integration challenges.
+To leverage the latest GenAI advancements effectively, developers need a framework that balances flexibility with standardization. 
+MLflow addresses this need with the :py:class:`mlflow.pyfunc.ChatModel` class introduced in 
+`version 2.11.0 <https://mlflow.org/releases/2.11.0#chatmodel-interface-for-a-unified-chat-experience-with-pyfunc-models>`_, providing a 
+consistent interface for GenAI applications while simplifying deployment and testing.
+
+Purpose of this tutorial
+------------------------
+
+This tutorial will guide you through the process of creating a custom chat agent using MLflow's :py:class:`mlflow.pyfunc.ChatModel` class. 
+
+By the end of this tutorial you will:
+
+- Integrate `MLflow Tracing <../tracing/index.html>`_ into a custom :py:class:`mlflow.pyfunc.ChatModel` instance.
+- Customize your model using the ``model_config`` parameter within :py:func:`mlflow.pyfunc.log_model`.
+- Leverage standardized signature interfaces for simplified deployment.
+- Recognize and avoid common pitfalls when extending the :py:class:`mlflow.pyfunc.ChatModel` class.
+
+.. contents::
+   :local:
+
+Prerequisites
+-------------
+
+- Familiarity with MLflow logging APIs and GenAI concepts.
+- MLflow version 2.11.0 or higher installed.
```

Comment:
since we mention tracing in the tutorial, should we mention that tracing is only available in 2.14.0 and above?

Reply:
great catch